code/go/0chain.net/smartcontract/interestpoolsc/sc.go

```diff
@@ -59,6 +62,24 @@ func (ipsc *InterestPoolSmartContract) GetRestPoints() map[string]smartcontracti
 	return ipsc.RestHandlers
 }
 
+func (ipsc *InterestPoolSmartContract) GetCost(t *transaction.Transaction, funcName string, balances c_state.StateContextI) int {
+	node := ipsc.getGlobalNode(balances, funcName)
```

Comment:
the name of node is conflict with the package node name, please rename it.

Reply:
fixed